src/plugins/inspector/public/views/data/components/__snapshots__/data_view.test.tsx.snap

```diff
@@ -235,8 +235,24 @@ exports[`Inspector Data View component should render empty state 1`] = `
         "timeZone": null,
       }
     }
+    kibana={
+      Object {
+        "notifications": Object {
+          "toasts": Object {
+            "danger": [Function],
+            "show": [Function],
+            "success": [Function],
+            "warning": [Function],
+          },
+        },
+        "overlays": Object {
+          "openFlyout": [Function],
+          "openModal": [Function],
+        },
+        "services": Object {},
+      }
+    }
```

Comment:
nit: It would be preferable if there were any way to update the unit tests so that these dependencies didn't appear in snapshots... otherwise they break any time future changes are made to the services, even if they're non-breaking

Reply:
I'll think how to refactor that test, and create a separate PR for that. I see here we just test a snapshot. It's not so useful...